docs/release-notes/release-notes-0.9.5.md

```diff
@@ -0,0 +1,190 @@
+
+<https://github.com/joinmarket-org/joinmarket-clientserver/issues>
+
+
+Upgrading
+=========
+
+To upgrade:
+
+*Reminder: always back up and recreate your joinmarket.cfg file when doing the upgrade; this is to make sure you have the new default settings.*
+
+(If you are upgrading from a version pre-0.7.0 please read the "Upgrading" section in [the 0.7.0 release notes](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.7.0.md).)
+
+(If you are upgrading from a version pre-0.9.0 please read the [release notes](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.9.0.md) to find out how about [fidelity bonds](../fidelity-bonds.md) are relevant to your use-case).
+
+First run the `install.sh` script as mentioned in the README. When prompted to overwrite the directory `jmvenv`, accept.
+
+Note that `./install.sh -?` will show the options for installation.
+
+Notable changes
+===============
+
+### Support sending to taproot
+
+This is mostly (though not 100%) a change to the backend bitcoin library python-bitcointx, which has been upgraded to [v1.1.3](https://github.com/Simplexum/python-bitcointx/releases/tag/python-bitcointx-v1.1.3) and in this version, includes the validation of bech32m as taproot addresses.
+(Note that the master branch of python-bitcointx now has full taproot support, i.e. constructing and spending taproot outputs, also; but we don't yet have a use case for that, anyway).
+Users can now send to taproot addresses via any of the user interfaces (CLI, Qt GUI, web UI).
+
+`b700f37` Add send-to-P2TR
+
+`a467ec4` upgrade to python-bitcointx 1.1.3
+
+### Extensive updates to the JSON RPC-API (jmwalletd.py); first fully working version
+
+After a lot of testing and bugfixing, the following commits were added to patch up the JSON RPC-API feature that was added in [0.9.3](https://github.com/JoinMarket-Org/joinmarket-clientserver/releases/tag/v0.9.3). The new [webUI project](https://github.com/joinmarket-webui/joinmarket-webui) is using this and is now functional (alpha release coming shortly).
+
+For those not familiar, the RPC API can be served by running the script `jmwalletd.py` and note that unlike other Joinmarket scripts, a wallet need not be specified, since it will allow the unlocking of any wallet (or creation of a new one) from a calling RPC client. See the [user docs](https://joinmarket-org.github.io/joinmarket-clientserver/JSON-RPC-API-using-jmwalletd.html) and [API docs](https://joinmarket-org.github.io/joinmarket-clientserver/api/) for details.
+
+There are several commits, broken into two groups here:
+
+#### New endpoints/functionality
+
+`29b459a` Add freeze method to RPC-API
+`89896e0` Add yieldgen report endpoint to API
+`6e07e4f` Add /taker/stop endpoint to RPC
+`28fdaa1` Allow re-unlock of wallets via /unlock
+`3939714` Add status, label and extradata to RPC display
+`e598b35` Add RPC API endpoint for showseed
```

Comment:
```suggestion
`29b459a` Add freeze method to RPC-API

`89896e0` Add yieldgen report endpoint to API

`6e07e4f` Add /taker/stop endpoint to RPC

`28fdaa1` Allow re-unlock of wallets via /unlock

`3939714` Add status, label and extradata to RPC display

`e598b35` Add RPC API endpoint for showseed
```

Reply:
This was a conscious choice (I know in the past it's been a common error). There are a lot of these commits, I felt it would take up far too much space and people maybe wouldn't even read the rest of the doc.

Reply:
They look like this when github formatted, was that intent? 
![image](https://user-images.githubusercontent.com/4500994/154531210-432d54ea-67a5-415d-bf05-183cc20fecec.png)

Reply:
Yes, I know. I did look at it (and have seen it before). It's maybe not the *best* way to present them in-line, but it's not the worst is it?

Reply:
Oh, that's a thought. The spacing of bullet points might be better yeah. I'll take a look.

Reply:
I'm going to open 2 new test PRs to compare. It's rather annoying but these markdown files *don't* render exactly the same either in gists, or in github comments, as they do in documents in the repo, so it's a little tricky.

Reply:
Just change in this PR, then look at "View file" under "Files changed" and revert if it does not look good?

Reply:
It's fine, done it. I don't think it's perfect in the sense that it does spam up the space still, but at least it's better. Will do that.